exercises/twelve-days/twelve-days.spec.js

```diff
@@ -0,0 +1,105 @@
+var TwelveDays = require('./twelve-days');
+
+describe('TwelveDays', function () {
+  var twelveDaysObject = new TwelveDays();
+
+  beforeEach(function () {
```

Comment:
If it's not needed, I would just delete it

Reply:
It doesn't seem as though it's possible to remove the new TwelveDays object on line 4. Even though line one calls the require function, a new instance would need to be made for the prototype members to be available. Any suggestions?

Reply:
@AdhirRamjiawan what we meant is that the `beforeEach` method is no longer needed.  You are correct that the `twelveDaysObject` must still be declared.

Sorry if that was unclear :)